meta/documents/user_guide/en/initial_setup.md

```diff
@@ -0,0 +1,130 @@
+# Wayfair plugin: Initial Setup
+
+## Prerequisites
+
+* [A Plentymarkets system](https://www.plentymarkets.co.uk).
+
+* Administrative rights on the Plentymarkets system where the Wayfair plugin will be used
+    - The Plentymarkets user's `Access` setting must be `Admin`
+    - The Plentymarkets user must be able to modify Plugin Sets
+
+* Active Wayfair supplier status
+    * A Wayfair Supplier ID is required
+    * [Information for prospective suppliers](https://partners.wayfair.com/d/onboarding/sell-on-wayfair)
+
+* [Wayfair API credentials](obtaining_credentials.md).
+
+* [Installation of the Wayfair plugin](plugin_installation.md).
+
+
+## 1. Authorizing the Wayfair Plugin to access Wayfair interfaces
+After the plugin is installed in your Plentymarkets Plugin Set, the plugin must be configured to use the correct credentials when connecting to Wayfair's interfaces.
+
+* **The authorization procedure must be performed for any Plugin Set that contains the Wayfair plugin**.
+* Copying a Plugin Set will copy the authorization information to the new plugin set.
+* An exported or imported Plugin Set may include the authorization information.
+
+The authorization steps are as follows:
+1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`
+
+2. Locate the Plugin Set that is linked to the client with which Wayfair will be used.
+
+3. Click on the `Edit` button for the desired Plugin set
+
+4. In the Wayfair row of the Plugin set, click on the `Settings` button.
+
+4. In the left-side menu, go to `Configuration` >> `Global Settings`.
+
+5. In the `Supplier Settings` area, enter the `Client ID` and `Client Secret` values that correspond with your Wayfair API credentials
+
+6. Change the `Mode` setting to `Live` - see [information on `Test` mode](test_mode.md)
+
+7. Click the `Save` button in the toolbar above the settings
+
+## 2. Activating the order referrer
+An order referrer in Plentymarkets identifies the sales channel on which an order was generated. To get the Plentymarkets system to properly import orders from the Wayfair API, the Wayfair order referrer must be activated:
+
+1. From the main Plentymarkets page, go to `Setup` >> `Orders` >> `Order referrer`.
+
+2. Place a check mark next to the **Wayfair** order referrer.

```

Comment:
I see two 'Wayfair' options as well as 'WayfairX' and 'WayfairY'. I'm assuming suppliers will only see one?

Reply:
Yeah `WayfairX` and `WayfairY` are artifacts that we created in the system for testing. Any supplier that's setting up more than one Wayfair supplier is doing it for their personal reasons and they're on their own.